utils/quest.ts

```diff
@@ -18,3 +18,26 @@ export function pickRandomObjectsFn({
     return questArray;
   }
 }
+
+export const getClaimableQuests = (
+  quests: QuestDocument[],
+  pendingBoostClaims: PendingBoostClaim[] | undefined
+) :ClaimableQuestDocument[] | undefined => {
+  if (pendingBoostClaims && pendingBoostClaims.length) {
```

Comment:
can write this as 
if(!pendingBoostClaims) return;
// rest of the code

helps with avoiding nested standalone if blocks